scripts/cnode-helper-scripts/prereqs.sh

```diff
@@ -163,9 +163,7 @@ if [ "$WANT_BUILD_DEPS" = 'Y' ]; then
     echo "CentOS: curl pkgconfig libffi-devel gmp-devel openssl-devel ncurses-libs ncurses-compat-libs systemd-devel zlib-devel tmux"
     err_exit
   fi
-  ghc_v=$(ghc --version | grep 8\.6\.5 2>/dev/null)
-  cabal_v=$(cabal --version | grep version\ 3 2>/dev/null)
-  if [ "${ghc_v}" = "" ] || [ "${cabal_v}" = "" ]; then
+  if ! ghc --version | grep -q 8\.10\.2 || ! cabal --version | grep -q version\ 3; then
```

Comment:
we should prolly change ghcup install  / set to 8.10.2 as well

Reply:
That was the intention, great that you noticed this missed change.